x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_data_streams_stats/index.ts

```diff
@@ -12,42 +12,31 @@ import { indexStatsService } from '../../../services';
 export async function getDataStreamsStats({
   esClient,
   dataStreams,
-  sizeStatsAvailable = true,
 }: {
   esClient: ElasticsearchClient;
   dataStreams: string[];
-  sizeStatsAvailable?: boolean; // Only Needed to determine whether `_stats` endpoint is available https://github.com/elastic/kibana/issues/178954
-}) {
+}): Promise<Record<string, { size: string; sizeBytes: number; totalDocs: number }>> {
   if (!dataStreams.length) {
-    return {
-      items: [],
-    };
+    return Promise.resolve({});
```

Comment:
The function is already async, there's no need for the extra Promise.resolve wrapping
```suggestion
    return {};
```